### ISSUE TITLE
line in TS source code is changed when looking at Pull request files diff 

### ISSUE THREAD

**Issue by capaj** [OctoLinker/OctoLinker#1165]
**Describe the bug**
see these screens: https://imgur.com/a/tPwgALV

**Expected behavior**
the line in github diff should be unchanged

**Additional context**
this line
```ts
import type { UserSessionContext } from '../../gql/user-session-types/UserSessionModelAsAdmin'

```
rendered as:
`import type'../../gql/user-session-types/UserSessionModelAsAdmin' { UserSessionContext } from `

**Comment by stefanbuck** [comment]
Thanks for filing this issue! This is definitely related to changes made in #1066. Prior this change OctoLinker wasn't be able to insert links in this diff scenario. I will try to fix this as fast as possible.

**Comment by stefanbuck** [comment]
Alright, I figured out the problem and I might found a solution for it. I just need to make sure this is not breaking anything else and come up with an end2end test for it. The problem is actually quite interesting : your path contains `types` which also appears as a substring in `import type ...`. This and the fact that `gql` is highlighted in the diff is causing this problem.

**Comment by stefanbuck** [comment]
This has been fixed in version 6.8.1. Thanks again for filling the issue and sorry for the inconvenience.
